packages/labs/gen-manifest/src/index.ts

```diff
@@ -316,6 +319,23 @@ const convertEvent = (event: Event): cem.Event => {
   };
 };
 
+const convertAttribute = (
+  reactiveProperty: ReactiveProperty
+): cem.Attribute => {
+  return {
+    name:
+      typeof reactiveProperty.attribute === 'string'
+        ? reactiveProperty.attribute
+        : reactiveProperty.name,
+    type: transformIfNotEmpty(reactiveProperty.type, convertType) ?? {
+      text: 'unknown',
+    },
+    ...convertCommonInfo(reactiveProperty),
+    default: reactiveProperty.default,
```

Comment:
I don't think this is _entirely_ correct, but it's ok for now. The attribute default is really the property default run through the property converter. This is probably the same most of the time, but will be different for arrays and objects and non-default converters.

Also, for boolean attributes this isn't quite right, as often it'll say the default is false.

Ideally attributes don't even have defaults, because reflecting initial values is bad. But we're in a spot where they do for now.

So how about if we add the default here if the only if:
- The property reflects
- The type is String, Number, or Boolean
- There is a property default
- There is no custom converter being used

Reply:
@justinfagnani thanks! Done in 7a88ea5